content/docs/concepts/metric_types.md

```diff
@@ -48,6 +48,10 @@ A _histogram_ samples observations (usually things like request durations or
 response sizes) and counts them in configurable buckets. It also provides a sum
 of all observed values.
 
+Since each observation bucket holds the total count of observations less than or
```

Comment:
This is an implementation detail, so would better go in the exposition format docs than the more user-facing docs.